ebos/equil/initstateequil.hh

```diff
@@ -1109,8 +1123,11 @@ private:
         std::vector<int> cellPvtRegionIdx(numCompressed);
 
         //Get the PVTNUM data
+#ifdef ENABLE_3DPROPS_TESTING
+        const auto pvtnumData = eclState.fieldProps().get_global<double>("PVTNUM");
```

Comment:
Why the copy? Even if a temporary is returned, one can extend its lifetime long enough by
```
const auto& pvtnumData = eclState.fieldProps().get_global<double>("PVTNUM");
```

Judging from the code below the vector has cartesian size like before (in contrast to other properties). That might come as a suprise.

Reply:
For both of these cases I have - initially - opted to use a global vector of cartesian size - just to keep the amount of changes down. The global vectors are by nature copies/temporaries, and then I like to make that clear by capturing the return value in a value object; I would assume that both the value object and the value held alive a with a const reference would have the lifetime - so memory wise it should make no difference?

But I'd be happy to change to `const &`- but maybe even more importantly is that this should eventually be a `get<double>( )` which returns a compressed vector instead of `get_global<double>( )`.